webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js

```diff
@@ -20,6 +21,14 @@ const ContentViewVersionContent = ({ cvId, versionId, cvVersion }) => {
     ansible_collection_count: ansibleCollectionCount,
   } = cvVersion;
 
+  const noCounts =
+    !Number(debCount) && !Number(dockerManifestCount) && !Number(dockerTagCount) &&
+    !Number(fileCount) && !Number(moduleStreamCount) && !Number(ansibleCollectionCount);
+
+  if (noCounts) {
+    return <InactiveText text={__('No content')} />;
+  }
+
```

Comment:
This doesn't take into account the generic content displayed which happens later in the file. There's code further down to show N/A, that can be modified.

Reply:
On it